### PR TITLE
docs(ui5-multi-input): add storybook sample

### PR DESCRIPTION
**Pull request by Okiana**
Add storybook sample for ui5-multi-input

### CODE REVIEW

File: packages/playground/_stories/main/MultiInput/MultiInput.stories.ts [hunk 1]
```diff
@@ -24,72 +26,86 @@ export default {
...
-	<div class="snippet">
-		<ui5-multi-input class="samples-margin samples-responsive-margin-bottom" value="basic input"></ui5-multi-input>
-		<ui5-multi-input class="samples-margin samples-responsive-margin-bottom" show-value-help-icon="" value="value help icon"></ui5-multi-input>
-	</div>
-`;
+export const BasicMultiInput = Template.bind({});
+BasicMultiInput.args = {
+	value: "basic input"
+};
+BasicMultiInput.storyName = "Basic Multi Input";
```
Comment (elenastoyanovaa) [inline packages/playground/_stories/main/MultiInput/MultiInput.stories.ts:48]: no need

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
- Change the PR name to docs, not feat.
- Remove the ts-ignore

File: packages/playground/_stories/main/MultiInput/MultiInput.stories.ts (hunk 1 above)
Comment (Okiana) [reply packages/playground/_stories/main/MultiInput/MultiInput.stories.ts:48]: Done :)

File: packages/playground/_stories/main/MultiInput/MultiInput.stories.ts [hunk 2]
```diff
@@ -24,72 +26,86 @@ export default {
...
+	<ui5-suggestion-item text="Sweden"></ui5-suggestion-item>
+	<ui5-suggestion-item text="USA"></ui5-suggestion-item>`,
+	valueStateMessage: '<div slot="valueStateMessage">Token is already in the list</div>',
+	showSuggestions: true
+};
+MultiInputTokenCreation.decorators = [
+	(story) => {
+		return html`
+		${story()}
 		<script>
```
[the remark this answers is not in the document]
Comment (Okiana) [reply packages/playground/_stories/main/MultiInput/MultiInput.stories.ts]: Done

**Comment by Okiana** [comment]
> * Change the PR name to docs, not feat.
> * Remove the ts-ignore

Done

**Review by elenastoyanovaa — APPROVED** [review]

**Pull request merged**